### PR TITLE
Feature/UI/framework/vue3

### PR DESCRIPTION
**Pull request by Helltab**
fix these issues #76 #106 #104 

1. this framework include these features: 
`Vue3` `Vite` `TypeScript` `Ant Design Vue` `AntV|echarts` `lodash` `i18n` `colorful theme`
2. more details are in ./ui-vue3/README.md file
image

### CODE REVIEW

File: ui-vue3/src/base/http/request.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        }
+        return config
+    },
+    (error) => {
+        Promise.reject(error)
+    }
+)
+
+response.use(
+    (response) => {
```
Comment (SDUWYS) [inline ui-vue3/src/base/http/request.ts:50]: Wouldn't it be better to add some global exception handling hints? 
Do we need to add login verification in the future?

**Review by SDUWYS — APPROVED** [review]
LGTM

File: ui-vue3/src/base/http/request.ts (hunk 1 above)
Comment (Helltab) [reply ui-vue3/src/base/http/request.ts:50]: please create a issue  about the  global exception and login

**Review by chickenlj — APPROVED** [review]
LGTM.

**Review by jianyi-gronk — COMMENTED** [review]
LGTM.

**Pull request merged**